opensrp-app/src/main/java/org/smartregister/repository/TaskRepository.java

```diff
@@ -710,10 +710,18 @@ public int getUnsyncedCreatedTasksAndTaskStatusCount() {
 
     @NonNull
     public Set<Task> getTasksByJurisdictionAndPlan(@NonNull String jurisdictionId, String planIdentifier) {
-        String query = "SELECT * FROM " + TASK_TABLE + " WHERE " + GROUP_ID + " = ? AND " + PLAN_ID + " = ?";
+        String query = TextUtils.join(" ",
+                new String[]{"SELECT * FROM", TASK_TABLE, "WHERE", GROUP_ID + "=?", "AND", PLAN_ID, "=?"});
         return getTasks(query, new String[]{jurisdictionId, planIdentifier});
     }
 
+    @NonNull
+    public Set<Task> getTasksByJurisdiction(@NonNull String jurisdictionId) {
```

Comment:
Preferred to add a new method that queries by `jurisdictionId` alone since a call to [this method](https://github.com/opensrp/opensrp-client-core/pull/785/files#diff-7dde074fe790b38f60f3c0431fcf46461c1ef850e99c8e7e2e31b725889e319dR712) with a null `planIdentifier`can still be considered a valid query that should only return Tasks with a null `planIdentifier` and which also match the specified `jurisdictionId`.